modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2528,12 +2528,27 @@
 
 	end
 
+	function m.emitPlatformVersion(optSdkVersion)
```

Comment:
This entire function can be reverted with the above change as it would just be a `nil` check.

Reply:
Yes, I think it can be a much smaller change. I took made the change as is from our local version. I didn't want to make changes to it at the same time.